doc/api/events.md

```diff
@@ -1166,6 +1166,68 @@ const emitter = new EventEmitter();
 setMaxListeners(5, target, emitter);
 ```
 
+## Class: `events.EventEmitterAsyncResource extends EventEmitter`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Integrates `EventEmitter` with {AsyncResource} for `EventEmitter`s that
+require manual async tracking.
```

Comment:
Maybe add some more description what this means. I assume that all events emitted via this emitter run with the asyncId of this emitter but I doubt that is clear for everyone.
Some example use case would be helpful.